catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml

```diff
@@ -220,8 +220,8 @@
             name="Summary Metacard Attributes"
             description="List of metacard attributes to display in the summary view."
             type="String"
-            cardinality="10000"
-            default="created,modified,thumbnail"
+            cardinality="10"
```

Comment:
The number itself is just chosen arbitrarily. The idea is to restrict how many attributes the users can add to under `Summary Metacard Attributes` inside `Catalog UI Search`. This is to prevent the UI being over-flooded with attribute data since each attribute is going inside where the Coordinates are being displayed inside Intrigue

Reply:
This might just be me, but I feel like this is abusing the summary config for an unintended purpose. It might be beneficial to use a separate config for the coordinate display so it doesn't negatively impact downstream projects and isn't coupled to the summary.